Sources/FirebaseFirestore/Query+Swift.swift

```diff
@@ -0,0 +1,36 @@
+// SPDX-License-Identifier: BSD-3-Clause
+
+@_exported
+import firebase
+@_spi(Internal)
+import FirebaseCore
+
+import CxxShim
+
+public typealias Query = firebase.firestore.Query
+
+extension Query {
+  var firestore: Firestore {
+    swift_firebase.swift_cxx_shims.firebase.firestore.query_firestore(self)
+  }
+
+  func getDocuments(completion: @escaping (QuerySnapshot?, Error?) -> Void) {
```

Comment:
I don't think that this is public API, is this variant needed (other than for use internally?)

```suggestion
  private func getDocuments(completion: @escaping (QuerySnapshot?, Error?) -> Void) {
```

Reply:
I meant for it to be `public`. Will fix.